api/client/ps/formatter.go

```diff
@@ -71,3 +76,65 @@ func tableFormat(ctx Context, containers []types.Container) {
 
 	customFormat(ctx, containers)
 }
+
+func customFormat(ctx Context, containers []types.Container) {
```

Comment:
@vdemeester did you just move this here or did you make some changes? (sorry lazy to check)